server/app/models/volume.rb

```diff
@@ -34,4 +34,16 @@ def name_for_service(service, instance_number)
     end
   end
 
+  def services
+    services = []
+    self.grid.grid_services.each do |service|
+      service.service_volumes.each do |sv|
```

Comment:
Something like `service.service_volumes.where(volume: self)`? Is there any direct way to query embedded documents?